src/Command/DatabaseCommand.php

```diff
@@ -291,22 +303,32 @@ private function generateCliParameters(DatabaseUrl $databaseUrl, bool $excludeDa
         return implode(' ', $result);
     }
 
-    private function excludeParts(string $database): string
-    {
-        $excludeOptions = [];
-        foreach ($this->getExcludes() as $exclude) {
-            $excludeOptions[] = '--ignore-table=' . escapeshellarg($database . '.' . $exclude);
-        }
-        if ($excludeOptions) {
-            return ' ' . implode(' ', $excludeOptions);
-        }
-        return '';
-    }
-
     private function getDatabaseCreateStatement(string $targetDatabase): string
     {
         $escapedTargetDatabase = '`' . str_replace('`', '``', $targetDatabase) . '`';
         $intermediateSql = sprintf('CREATE DATABASE IF NOT EXISTS %s;USE %s;', $escapedTargetDatabase, $escapedTargetDatabase);
         return ' | (echo ' . escapeshellarg($intermediateSql) . ' && cat)';
     }
+
+    /**
+     * @param string[] $excludes
+     * @return string
+     */
+    private function getExcludeSqlPart(array $excludes): string
+    {
+        $result = '';
+        $simpleExclude = [];
+        foreach ($excludes as $exclude) {
+            $stringLength = strlen($exclude);
+            if ($stringLength > 3 && $exclude[0] === '/' && $exclude[$stringLength - 1] === '/') {
```

Comment:
did you document, that regex requires at least 3 letters and has to start with / to work?

the psr for str_starts_with would have been super helpful here :D 
And btw, if we might need it somewhere else, it might not be stupid to consider putting it into the StringHelper

Reply:
/a/ are 3characters
This is absolutely not documented, unless in the readme